src/mlpack/tests/loss_functions_test.cpp

```diff
@@ -871,5 +872,44 @@ BOOST_AUTO_TEST_CASE(SoftMarginLossTest)
   BOOST_REQUIRE_EQUAL(output.n_cols, input.n_cols);
   CheckMatrices(output, expectedOutput, 0.1);
 }
+/**
+ * Simple test for the Mean Absolute Percentage Error function.
+ */
+BOOST_AUTO_TEST_CASE(MeanAbsolutePercentageErrorTest)
+{
+  arma::mat input, target, output, expectedOutput;
+  MeanAbsolutePercentageError<> module;
+
+  // Test for sum reduction.
+  input = arma::mat("3, -0.5, 2, 7");
+  target = arma::mat("2.5, 0.2, 2, 8");
+  expectedOutput = arma::mat("0.4 5.0 0.5 0.125");
+
+  // Test the Forward function. Loss should be 3.825.
```

Comment:
Add a comment saying that the value has been calculated manually both for this and the comment on line 903.